### PR TITLE
Tweaks Sleeping Carp a bit and adds a new bundle with Sleeping Carp

### PR DESCRIPTION
**Pull request by keronshb**
## About The Pull Request

This tweaks Sleeping Carp a bit to deal with stamina a TINY bit more but not a lot.  It gives them a parry also.

There's also a separate bundle that gives you the Bo Staff and a Carp Costume.

Huge thanks to Augments, Kevin, and Teak for helping me through this.

## Why It's Good For The Game

SCarp is in a very lackluster place after taking a few nerfs and it can't deal with stamina combat that much.  Despite being stun resistant, it isn't that much stamina resistant.  So the changes should reflect the cost of TC now.  

## Changelog
:cl:
add: Adds a reduced stamina buffer for SCarp users
add: Gives SCarp users a better parry
add: Adds the SCarp bundle which includes a bo staff
balance: reduces the stamina damage of scarp slightly
balance: reduced the blockchance of the bo staff
/:cl:

### CODE REVIEW

File: code/modules/vehicles/cars/clowncar.dm [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
 	initialize_controller_action_type(/datum/action/vehicle/sealed/horn/clowncar, VEHICLE_CONTROL_DRIVE)
 
+/*
+/obj/vehicle/clowncar/vehicle_move(direction) //Prevent it from moving onto space
+	if(isspaceturf(get_step(src, direction)))
+		return FALSE
+	else
+		return ..()
+*/
+
```
Comment (timothyteakettle) [inline code/modules/vehicles/cars/clowncar.dm:28]: if this isn't relevant to the PR, do not add it

File: code/modules/vehicles/cars/clowncar.dm [hunk 2]
```diff
@@ -56,7 +64,7 @@
 		L.visible_message("<span class='warning'>[src] rams into [L] and sucks him up!</span>") //fuck off shezza this isn't ERP.
 		mob_forced_enter(L)
 		playsound(src, pick('sound/vehicles/clowncar_ram1.ogg', 'sound/vehicles/clowncar_ram2.ogg', 'sound/vehicles/clowncar_ram3.ogg'), 75)
-	else if(istype(M, /turf/closed))
+	else if(istype(M, /turf/closed) || istype(M, /obj/machinery/door/airlock/external))
```
Comment (timothyteakettle) [inline code/modules/vehicles/cars/clowncar.dm:67]: same thing here

Comment (keronshb) [reply code/modules/vehicles/cars/clowncar.dm:67]: FUCK how did these get by.

File: code/modules/vehicles/cars/clowncar.dm (hunk 1 above)
Comment (keronshb) [reply code/modules/vehicles/cars/clowncar.dm:28]: F wrong button there.  I don't know how to remove these.

Comment (Putnam3145) [reply code/modules/vehicles/cars/clowncar.dm:28]: Just edit it again.

**Comment by keronshb** [comment]
This fucked up clearly

File: code/modules/vehicles/cars/clowncar.dm (hunk 1 above)
Comment (Putnam3145) [reply code/modules/vehicles/cars/clowncar.dm:28]: Like... you can edit the open PR the same way you edited it to open it in the first place.

**Pull request closed without merging**